docs/fastdds/rpc_dds/rpc_dds_intro.rst

```diff
@@ -6,18 +9,43 @@ RPC over DDS
 .. _RPC over DDS specification: https://www.omg.org/spec/DDS-RPC/1.0/PDF
 
 *Remote Procedure Calls* (see `RPC over DDS specification`_), also known as RPC,
-is a type of bidirectional communication used in a request-reply pattern.
+is a type of bidirectional communication, used in a request-reply pattern, that can be implemented
+using DDS entities such as DomainParticipants, Topics, DataWriters and DataReaders.
+
 The RPC architecture is based on the client-server model: the client sends a request to the server,
-and the server sends one or more responses (replies) back to the client.
-In the context of Data Distribution Service (DDS), the client and server are modeled using DomainParticipants
-and they contain all the necessary objects to communicate with each other.
-Each client (internally represented by a *Requester* entity) sends a request sample
-to the server through a request topic;
-the server (internally represented by a *Replier* entity) processes it and sends a reply sample back to the client
-through a different topic (reply topic).
-All the Requesters and Repliers internally have a DataWriter and DataReader to send and receive samples.
-
-.. toctree::
-    :maxdepth: 2
-
-    /fastdds/rpc_dds/request_reply_api/request_reply_api_intro
+and the server sends one response (reply) back to the client.
+
+The RPC specification provides two different APIs to build RPC over DDS applications:
+
+* *A low-level request/reply API* (see :ref:`request_reply_api_intro`), which is based on the DDS API.
+  Clients and Servers are modeled by a *Requester* or *Replier* entities, respectively,
+  and created from a DomainParticipant.
+
+  Each *Requester* and *Replier* contains a DataWriter and DataReader to send and receive samples.
+  Requester sends request samples to the the Replier through a request topic,
+  and the Replier, after processing the data and computing the result of the operation,
+  sends reply samples back to the Requester through a different topic (reply topic).
+
+  This API is intended to be used internally by the *Fast DDS-Gen* tool, so it is not recommended
+  to be used directly by the user.
+
+* *A high-level function-call style based API*, built on top of the request/reply API, and generated automatically
```

Comment:
Add a short paragraph here introducing the streaming feature and a link to the page where is described.